packs/data/equipment-effects.db/effect-owlbear-claw.json

```diff
@@ -0,0 +1,52 @@
+{
+    "_id": "LHrEiMzmjU6NRQSw",
+    "img": "systems/pf2e/icons/equipment/consumables/talismans/owlbear-claw.webp",
+    "name": "Effect: Owlbear Claw",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Owlbear Claw]{Owlbear Claw}</p>\n<p>The triggering attack gains the weapon's critical specialization effect.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": {
+                    "ownedItems": true,
+                    "types": [
+                        "weapon"
+                    ]
+                },
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Weapon"
+            },
+            {
+                "key": "CriticalSpecialization"
```

Comment:
Do you need to predicate this based on the ChoiceSet?

Reply:
yes, pushed the wrong version of this. only valid for the weapon they affixed to.

Reply:
however, since it only gets toggled on the actor after they rolled the crit attack and before crit damage is rolled, it's functional without that at the moment. I'm now rethinking whether the ChoiceSet even matters in this workflow.